config/default_records/tag_groups/005_ultima.yml

```diff
@@ -0,0 +1,199 @@
+Ultima P1:
```

Comment:
Little concerned about the group name being mostly the sequencing machine name. If Ultima sequencing machines get used a lot in future, and more tags are created, this name could prove to be way too non-descriptive. Normally the sequencer is not specified for tag groups.
However, there doesn't seem to be much of a standard.
e.g. PF_adapters_UDI_96_A_i7
e.g. PN-1000212 Single Index Kit N, Set A
e.g. pWGS i5 UDI tag group v2 A

Reply:
Rich Clark has been advocating for using the exact manufacturer names (e.g. 'PN-1000212 Single Index Kit N, Set A') - but I believe Sabrine suggested this already and they prefer this naming.